Moesif.Middleware/Moesif.Middleware.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>

```

Comment:
care to share background on what necessary to change this? the `README.md` still specifies its supported on `4.5x` - do we no longer support.. should then update readme too.

Reply:
yes.  It is due to this ticket.
https://github.com/microsoft/ApplicationInsights-dotnet/issues/1312

Reply:
if minimum support is changing.. please update `README.md` in this repo too to remove `4.5`.